.github/workflows/coverage.yml

```diff
@@ -0,0 +1,64 @@
+name: Code Coverage
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+
+jobs:
+  code-coverage:
+    name: Code Coverage
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.81.0
+          components: llvm-tools-preview
+
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+
+      - run: |
+          sudo apt remove -y postgres*
+          sudo apt-get install -y wget gnupg
+          sudo sh -c 'echo "deb https://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt-get update -y -qq --fix-missing
+          sudo apt-get install -y \
+            clang-10 \
+            llvm-10 \
+            clang \
+            gcc \
+            make \
+            build-essential \
+            libz-dev \
+            zlib1g-dev \
+            strace \
+            libssl-dev \
+            pkg-config \
+            postgresql-16 \
+            postgresql-server-dev-16
+          sudo chmod a+rwx `/usr/lib/postgresql/16/bin/pg_config --pkglibdir` `/usr/lib/postgresql/16/bin/pg_config --sharedir`/extension /var/run/postgresql/
+
+      - run: cargo install cargo-pgrx --version 0.12.6
+      - run: cargo pgrx init --pg16 /usr/lib/postgresql/16/bin/pg_config
```

Comment:
Will it matter if this code coverage check is only done on pg 16 but that our targets are pg15 and 17 downstream?

Reply:
No, it shouldn't matter because this is just generating code coverage which for all practical purposes will be the same whichever pg version we use here.